component/src/main/java/org/wso2/extension/siddhi/gpl/execution/streamingml/classification/hoeffdingtree/HoeffdingClassifierUpdaterStreamProcessorExtension.java

```diff
@@ -0,0 +1,457 @@
+                        type = {DataType.DOUBLE, DataType.INT})
+        },
+        returnAttributes = {
+                @ReturnAttribute(name = "accuracy",
+                        description = "The accuracy evaluation of the model(Prequnetial Evaluation)",
+                        type = {DataType.DOUBLE})
+        },
+        examples = {
+                @Example(
+                        syntax = "define stream StreamA (attribute_0 double, attribute_1 double, "
+                                + "attribute_2 double, attribute_3 double, attribute_4 string );\n"
+                                + "\n"
+                                + "from StreamA#streamingml:updateHoeffdingTree('model1', 3) \n"
+                                + "select attribute_0, attribute_1, attribute_2, attribute_3, "
+                                + "accuracy insert into outputStream;",
+                        description = "A HoeffdingTree model with the name 'model1' will be built/updated under "
+                                + "3 number of classes using attribute_0, attribute_1, attribute_2, attribute_3 "
+                                + "as features and attribute_4 as the label. The accuracy evaluation will be "
+                                + "emitted to the outputStream"
+                ),
+                @Example(
+                        syntax = "define stream StreamA (attribute_0 double, attribute_1 double, "
+                                + "attribute_2 double, attribute_3 double, attribute_4 string );\n"
+                                + "\n"
+                                + "from StreamA#streamingml:updateHoeffdingTree('model1', 3, 200, 0, 1e-7, 1.0D, "
+                                + "true, true, 2) \n"
+                                + "select attribute_0, attribute_1, attribute_2, attribute_3, "
+                                + "accuracy insert into outputStream;",
+                        description = "A Hoeffding Tree model with the name 'model1' will be "
+                                + "built/updated with a grace period of 200, InformationGain Split criterion,"
+                                + "1e-7 of allowable error in split decision, 1.0D of breaktie threshold, "
+                                + "allowing only binary splits, disabled pre-pruning, Naive Bayes Adaptive as"
+                                + "leaf prediction using attribute_0, attribute_1, attribute_2, attribute_3 as "
+                                + "features and attribute_4 as the label. The accuracy evaluation will be "
+                                + "emitted to the outputStream."
+                )
+        }
+)
+public class HoeffdingClassifierUpdaterStreamProcessorExtension extends StreamProcessor {
+
+    private static final Logger logger = Logger.getLogger(HoeffdingClassifierUpdaterStreamProcessorExtension.class);
+
+    private static final int minNoOfFeatures = 3;
```

Comment:
Better to use all caps for the constants.

Reply:
Fixed in [#24fd7b0cd55c5a5589ddf0093a3c3fbe277252f9](https://github.com/wso2-extensions/siddhi-gpl-execution-streamingml/pull/16/commits/24fd7b0cd55c5a5589ddf0093a3c3fbe277252f9)